.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-*   @contentful/team-phoenix
-lib/cmds/init @contentful/team-core-experience
```

Comment:
are we sure we can remove it? 👀

Reply:
I think we should keep the path based ownership for the core-experience team.

Reply:
adjusted it